src/data/STATUSES/root/PLD.ts

```diff
@@ -116,4 +116,10 @@ export const PLD = ensureStatuses({
 		icon: 'https://xivapi.com/i/012000/012518.png',
 		duration: 21000,
 	},
+	BLADE_OF_FAITH_READY: {
+		id: 3019,
+		name: 'Blade of Faith Ready',
+		icon: 'https://xivapi.com/i/012000/012519.png',
+		duration: 30000,
+	},
```

Comment:
This doesn't exist in the root for the expac, it should be stubbed in root (so like `BLADE_OF_FAITH_READY: SHARED.UNKNOWN` and import as needed, then patch the details in the layer.